### ISSUE TITLE
LatestPosts Always Returns No Results

### ISSUE THREAD

**Issue by gordonbanderson** [silverstripe/silverstripe-forum#184]
The paramemters on this line https://github.com/silverstripe/silverstripe-forum/blob/0.8/code/controllers/ForumMemberProfile.php#L77 are the wrong way around

```
function LatestPosts() {
		return Post::get()
			->filter('AuthorID', (int)$this->urlParams['ID'])
			->limit(0,10) // <<<<<<<<<<<< should be 10,0
			->sort('Created', 'DESC')
			->filterByCallback(function($post){
				return $post->canView();
			});
 	}
```

**Comment by gordonbanderson** [comment]
Which branch to put this fix against?

**Comment by camfindlay** [comment]
This would be a bug fix in the 0.8 branch if you can make a PR I can merge this into 0.8, release a new 0.8.2 and also merge this upward into the new clean master.
